lib/phoenix_html/safe.ex

```diff
@@ -70,4 +70,11 @@ end
 
 defimpl Phoenix.HTML.Safe, for: Tuple do
   def to_iodata({:safe, data}), do: data
```

Comment:
> Maybe that's going away when the book is updated (if it hasn't already), I dunno.

It will be this week.

> Let me know where you'd like this to go and I'll make the change. 

Would you send a change that at least raises Protocol.UndefinedError for now without any extra description? Then it is just a matter of adding the description if the problem persists. :)